code/ZianeMaamar.php

```diff
@@ -0,0 +1,11 @@
+git add ZianeMaamar.php<?php
```

Comment:
Haha oops, i can see what happened here! :P

Reply:
Can you explain Im new to all this 
Am I Wrong ?? or I shouldn't make files in code ?? 
Thank you

Reply:
`git add ZianeMaamar.php<?php` it appears you've saved a command into your file by accident.

Reply:
`git add ZianeMaamar.php`

Reply:
Thank you and sorry i didn't see it I was blind Thank you again <3